### DISCUSSION TITLE
Facebook正式更名「Meta」

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#144] [General]
Facebook創辦人Mark Zuckerberg表示會更名Facebook母公司為Meta，並將Facebook名稱降為旗下產品品牌之一，與Instagram及WhatsApp並列。
Meta公司將會聚焦於強調虛擬實境與現實生活整合的元宇宙（Metaverse）目標，而不僅限於網路社群媒體，而具備元宇宙題材的不只有虛擬實境（VR）及擴增實境（AR），包括人工智慧、晶圓代工、社群平台、遊戲、金融都將扮演關鍵角色。

新聞網址
https://3c.ltn.com.tw/news/46473

**Comment by enoch-lightning** [comment]
元宇宙(metaverse)確實是勢在必行。我有找到美國NBC關於此新聞的報導。
[Facebook changes company name to Meta](https://www.cnbc.com/2021/10/28/facebook-changes-company-name-to-meta.html)

**Comment by Joan0518** [comment]
<img width="338" alt="擷取" src="https://user-images.githubusercontent.com/90574174/140070391-6b3dba25-bbfa-4af5-a5ef-c93780e797e2.PNG">
